### PR TITLE
Fix validation for edit repo authorization

### PR DESCRIPTION
**Pull request by CjayBillones**
Changed the validation checking in able_to_edit_repo? in user model.

### CODE REVIEW

**Comment by CjayBillones** [comment]
fixed bug

**Comment by CjayBillones** [comment]
Fix for #347

**Comment by schneems** [comment]
Seems good, thanks

File: app/models/user.rb [hunk 1]
```diff
@@ -60,7 +60,7 @@ def enqueue_inactive_email
   end
 
   def able_to_edit_repo?(repo)
-    repo.user_name == github
+    repo.user_name == github.downcase
```
Comment (prathamesh-sonpatki) [inline app/models/user.rb]: should both values be downcased?

Comment (prathamesh-sonpatki) [inline app/models/user.rb]: Ah. My bad. Didn't read issue properly :smile:

Comment (CjayBillones) [inline app/models/user.rb]: When I checked the code and what's being stored in the database, I observed that the repo.user_name is extracted from github and I think it's already converted to downcase. You might want to double check it to confirm.

Comment (CjayBillones) [inline app/models/user.rb]: It's okay. Thanks!

And by the way, this line of code from the Repo model might be the ones responsible in making the username and name downcased. :)

screenshot from 2015-06-13 02 37 43

Comment (prathamesh-sonpatki) [inline app/models/user.rb]: :heart:

**Pull request merged**